core/src/main/scala/org/apache/spark/ui/exec/ExecutorsTab.scala

```diff
@@ -157,4 +158,42 @@ class ExecutorsListener(storageStatusListener: StorageStatusListener, conf: Spar
     }
   }
 
+  private def updateExecutorBlacklist(eid: String, isBlacklisted: Boolean): Unit = {
+    val execTaskSummary = executorToTaskSummary.getOrElseUpdate(eid, ExecutorTaskSummary(eid))
+    execTaskSummary.isBlacklisted = isBlacklisted
+  }
+
+  override def onExecutorBlacklisted(executorBlacklisted: SparkListenerExecutorBlacklisted)
+  : Unit = synchronized {
```

Comment:
I know the Spark project has some style standards. It would be great if these could be expressed programmatically, and if failing to meet these standards caused a compile time error.